Cannot add exclusions more than once

**Build scan link**
Not applicable

**Plugin version**
1.18.0

**Gradle version**
7.4.2

**(Optional) Android Gradle Plugin (AGP) version**
Not applicable

**`reason` output for bugs relating to incorrect advice**
Not applicable

**Describe the bug**
Unable to declare 

**To Reproduce**
When having exclusions defined in multiple places
```
dependencyAnalysis {
  issues {
    onUnusedDependencies {
      exclude("commons-io:commons-io")
    }
  }
}
dependencyAnalysis {
  issues{
    onUnusedDependencies {
      exclude("org.apache.commons:commons-math3")
    }
  }
}
```
you will get an `IllegalStateException`:
```
Caused by: java.lang.IllegalStateException: The value for this property cannot be changed any further.
	at org.gradle.api.internal.provider.AbstractProperty$NonFinalizedValue.beforeMutate(AbstractProperty.java:386)
	at org.gradle.api.internal.provider.AbstractProperty.assertCanMutate(AbstractProperty.java:263)
	at org.gradle.api.internal.provider.AbstractProperty.setSupplier(AbstractProperty.java:212)
	at org.gradle.api.internal.provider.AbstractCollectionProperty.set(AbstractCollectionProperty.java:142)
	at com.autonomousapps.extension.Issue.exclude(IssueHandler.kt:329)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at org.gradle.internal.metaobject.BeanDynamicObject$MetaClassAdapter.invokeMethod(BeanDynamicObject.java:484)
	at org.gradle.internal.metaobject.BeanDynamicObject.tryInvokeMethod(BeanDynamicObject.java:196)
	at org.gradle.internal.metaobject.CompositeDynamicObject.tryInvokeMethod(CompositeDynamicObject.java:98)
	at org.gradle.internal.extensibility.MixInClosurePropertiesAsMethodsDynamicObject.tryInvokeMethod(MixInClosurePropertiesAsMethodsDynamicObject.java:34)
	at org.gradle.internal.metaobject.ConfigureDelegate.invokeMethod(ConfigureDelegate.java:56)
	at build_91dm6gpg3gc1uxkiyi2m5wlij$_run_closure4$_closure7$_closure8.doCall(/Users/daans/workspace/daanschipper/dependency-analysis-android-gradle-plugin/build/functionalTest/DoubleExclusionsProject-87515f8f-cd64-48-12/proj/build.gradle:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at org.gradle.util.internal.ClosureBackedAction.execute(ClosureBackedAction.java:73)
	at org.gradle.util.internal.ConfigureUtil.configureTarget(ConfigureUtil.java:155)
	at org.gradle.util.internal.ConfigureUtil.configure(ConfigureUtil.java:106)
	at org.gradle.util.internal.ConfigureUtil$WrappedConfigureAction.execute(ConfigureUtil.java:167)
	at com.autonomousapps.extension.ProjectIssueHandler.onUnusedDependencies(IssueHandler.kt:245)
	at com.autonomousapps.extension.ProjectIssueHandler_Decorated.onUnusedDependencies(Unknown Source)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at org.gradle.internal.metaobject.BeanDynamicObject$MetaClassAdapter.invokeMethod(BeanDynamicObject.java:484)
	at org.gradle.internal.metaobject.BeanDynamicObject.tryInvokeMethod(BeanDynamicObject.java:196)
	at org.gradle.internal.metaobject.CompositeDynamicObject.tryInvokeMethod(CompositeDynamicObject.java:98)
	at org.gradle.internal.extensibility.MixInClosurePropertiesAsMethodsDynamicObject.tryInvokeMethod(MixInClosurePropertiesAsMethodsDynamicObject.java:34)
	at org.gradle.internal.metaobject.ConfigureDelegate.invokeMethod(ConfigureDelegate.java:56)
	at build_91dm6gpg3gc1uxkiyi2m5wlij$_run_closure4$_closure7.doCall(/Users/daans/workspace/daanschipper/dependency-analysis-android-gradle-plugin/build/functionalTest/DoubleExclusionsProject-87515f8f-cd64-48-12/proj/build.gradle:22)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at org.gradle.util.internal.ClosureBackedAction.execute(ClosureBackedAction.java:73)
	at org.gradle.util.internal.ConfigureUtil.configureTarget(ConfigureUtil.java:155)
	at org.gradle.util.internal.ConfigureUtil.configure(ConfigureUtil.java:106)
	at org.gradle.util.internal.ConfigureUtil$WrappedConfigureAction.execute(ConfigureUtil.java:167)
	at com.autonomousapps.extension.IssueHandler.project(IssueHandler.kt:55)
	at com.autonomousapps.DependencyAnalysisSubExtension.issues(DependencyAnalysisSubExtension.kt:46)
```

**Expected behavior**
Being able to configure the exclusions in the extension more than once.

**Additional context**
This is especially useful when configuring the dependency analysis extension from plugins.

--- a/src/functionalTest/groovy/com/autonomousapps/jvm/DoubleExclusionsSpec.groovy
+++ b/src/functionalTest/groovy/com/autonomousapps/jvm/DoubleExclusionsSpec.groovy
@@ -1,0 +1,24 @@
+package com.autonomousapps.jvm
+
+import com.autonomousapps.jvm.projects.DoubleExclusionsProject
+
+import static com.autonomousapps.utils.Runner.build
+import static com.google.common.truth.Truth.assertThat
+
+final class DoubleExclusionsSpec extends AbstractJvmSpec {
+
+    def "project can declare exclusion twice (#gradleVersion)"() {
+        given:
+        def project = new DoubleExclusionsProject()
+        gradleProject = project.gradleProject
+
+        when:
+        build(gradleVersion, gradleProject.rootDir, "buildHealth")
+
+        then:
+        assertThat(project.actualProjectAdvice().isEmpty())
+
+        where:
+        gradleVersion << gradleVersions()
+    }
+}

--- a/src/functionalTest/groovy/com/autonomousapps/jvm/projects/DoubleExclusionsProject.groovy
+++ b/src/functionalTest/groovy/com/autonomousapps/jvm/projects/DoubleExclusionsProject.groovy
@@ -1,0 +1,74 @@
+package com.autonomousapps.jvm.projects
+
+import com.autonomousapps.AbstractProject
+import com.autonomousapps.kit.GradleProject
+import com.autonomousapps.kit.Plugin
+import com.autonomousapps.kit.Source
+import com.autonomousapps.kit.SourceType
+import com.autonomousapps.model.ProjectAdvice
+
+import static com.autonomousapps.AdviceHelper.actualProjectAdvice
+import static com.autonomousapps.kit.Dependency.commonsIO
+import static com.autonomousapps.kit.Dependency.commonsMath
+
+final class DoubleExclusionsProject extends AbstractProject {
+
+    private final javaLibrary = [Plugin.javaLibraryPlugin]
+
+    final GradleProject gradleProject
+
+    DoubleExclusionsProject() {
+        this.gradleProject = build()
+    }
+
+    private GradleProject build() {
+        def builder = newGradleProjectBuilder()
+        builder.withSubproject('proj') { s ->
+            s.sources = sources
+            s.withBuildScript { bs ->
+                bs.plugins = javaLibrary
+                bs.dependencies = [commonsIO('implementation'), commonsMath('implementation')]
+                bs.additions = """\
+          dependencyAnalysis {
+            issues { 
+              onUnusedDependencies {
+                exclude("commons-io:commons-io")
+              }
+            }
+          }
+          dependencyAnalysis {
+            issues{
+              onUnusedDependencies {
+                exclude("org.apache.commons:commons-math3")
+              }
+            }
+          }
+        """.stripIndent()
+            }
+        }
+
+        def project = builder.build()
+        project.writer().write()
+        return project
+    }
+
+    private sources = [
+            new Source(
+                    SourceType.JAVA, 'Main', 'com/example',
+                    """\
+        package com.example;
+       
+        public class Main {
+          public Main() {}
+        
+          public void hello() {
+            System.out.println("hello");
+          }
+        }
+      """.stripIndent())
+    ]
+
+    Set<ProjectAdvice> actualProjectAdvice() {
+        return actualProjectAdvice(gradleProject)
+    }
+}

--- a/src/main/kotlin/com/autonomousapps/extension/IssueHandler.kt
+++ b/src/main/kotlin/com/autonomousapps/extension/IssueHandler.kt
@@ -325,8 +325,7 @@ open class Issue @Inject constructor(objects: ObjectFactory) {
    * tells the plugin to exclude those dependencies in the final advice.
    */
   fun exclude(vararg ignore: String) {
-    excludes.set(ignore.toSet())
-    excludes.disallowChanges()
+    excludes.addAll(ignore.toSet())
   }
 
   internal fun behavior(): Provider<Behavior> {

